ZTNA APIs [Public Preview] (#21578)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/networkaccess-enrichedauditlogs.md

---
title: "enrichedAuditLogs resource type"

> description: "Represents a complete overview of the enriched audit logs workloads within the tenant. It consists of three partners: SharePoint, Teams, and Exchange. Each partner represents a specific workload that generates audit logs"

Reviewer: ```suggestion
description: "Represents a complete overview of the enriched audit logs workloads within the tenant. It consists of three partners: SharePoint, Teams, and Exchange. Each partner represents a specific workload that generates audit logs."
```

author: "Moti-ba"
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: resourcePageType
---

# enrichedAuditLogs resource type

Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents a complete overview of the enriched audit logs workloads within the tenant. It consists of three partners: SharePoint, Teams, and Exchange. Each partner represents a specific workload that generates audit logs

Reviewer: ```suggestion
Represents a complete overview of the enriched audit logs workloads within the tenant. It consists of three partners: SharePoint, Teams, and Exchange. Each partner represents a specific workload that generates audit logs.
```

Inherits from [microsoft.graph.entity](../resources/entity.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Get enrichedAuditLogs](../api/networkaccess-enrichedauditlogs-get.md)|[microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md)|Read the properties and relationships of a [microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md) object.|
|[Update enrichedAuditLogs](../api/networkaccess-enrichedauditlogs-update.md)|[microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md)|Update the properties of a [microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |exchange|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Exchange Online enriched audit logs settings	|
> |id|String|Id Inherited from [microsoft.graph.entity](../resources/entity.md).|
> |sharepoint|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|SharePoint Online enriched audit logs settings|
> |teams|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Teams enriched audit logs settings|

Reviewer: ```suggestion
|exchange|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Exchange Online enriched audit logs settings.    |
|id|String|Id Inherited from [microsoft.graph.entity](../resources/entity.md).|
|sharepoint|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|SharePoint Online enriched audit logs settings.|
|teams|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Teams enriched audit logs settings.|
```

---
title: "enrichedAuditLogs resource type"

> description: "This entity is designed to offer a comprehensive snapshot of our enhanced audit log workloads. At present, it encompasses three partners: SharePoint, Teams, and Exchange. Within each partner, the status indicates whether the corresponding feature is activated or deactivated"

Reviewer: Make it something like "_Defines whether enriched audit logs are available for the supported services that have corresponding traffic profiles in the Global Secure Access service." 

Then in the description section below:
1. add a base description like the one you apply in this metadata
2. And then explain what "enriched logs" are and how they compare with the networkaccess-logs.md, i.e. how do they look like in comparison with "basic" audit logs?

author: "Moti-ba"
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: resourcePageType
---

# enrichedAuditLogs resource type

Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> This entity is designed to offer a comprehensive snapshot of our enhanced audit log workloads. At present, it encompasses three partners: SharePoint, Teams, and Exchange. Within each partner, the status indicates whether the corresponding feature is activated or deactivated

Reviewer: See the comment in the metadata.
Also call out that enriched audit logs are supported for Microsoft SharePoint, Microsoft Teams, and Microsoft Exchange services.

|[Create enrichedAuditLogs](../api/networkaccess-settings-post-enrichedauditlogs.md)|[microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md)|Create a new [microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md) object.|
|[Get enrichedAuditLogs](../api/networkaccess-enrichedauditlogs-get.md)|[microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md)|Read the properties and relationships of a [microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md) object.|
|[Update enrichedAuditLogs](../api/networkaccess-enrichedauditlogs-update.md)|[microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md)|Update the properties of a [microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md) object.|

> |[Delete enrichedAuditLogs](../api/networkaccess-settings-delete-enrichedauditlogs.md)|None|Delete a [microsoft.graph.networkaccess.enrichedAuditLogs](../resources/networkaccess-enrichedauditlogs.md) object.|

Reviewer: Only GET and UPDATE are supported.

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |exchange|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Exchange Online enriched audit logs settings	|
> |id|String|Id Inherited from [microsoft.graph.entity](../resources/entity.md).|
> |sharepoint|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|SharePoint Online enriched audit logs settings|
> |teams|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Teams enriched audit logs settings|

Reviewer: ```suggestion
|exchange|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)| Defines whether enriched audit logs for Exchange Online is enabled.	|
|id|String|Id Inherited from [microsoft.graph.entity](../resources/entity.md).|
|sharepoint|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Defines whether enriched audit logs for SharePoint Online is enabled.|
|teams|[microsoft.graph.networkaccess.enrichedAuditLogsSettings](../resources/networkaccess-enrichedauditlogssettings.md)|Defines whether enriched audit logs for Teams is enabled.|
```
